src/commands/game_commands/play.ts

```diff
@@ -21,24 +28,65 @@ export default class PlayCommand implements BaseCommand {
             if (!voicePermissionsCheck(message)) {
                 return;
             }
+            const msgHasElimination = parsedMessage.components.length >= 1 && parsedMessage.components[0].toLowerCase() === "elimination";
             if (!gameSessions[message.guildID]) {
                 const textChannel = message.channel;
-                const gameSession = new GameSession(textChannel, voiceChannel, message.author);
+                const gameOwner = message.author;
+                const lives = parsedMessage.components.length > 1 ? parseInt(parsedMessage.components[1], 10) : DEFAULT_LIVES;
```

Comment:
These elimination specific values should be instantiated if we're in elimination mode